runtime/mozart/Cargo.toml

```diff
@@ -0,0 +1,102 @@
+[package]
+name = "tanssi-relay"
+default-run = "tanssi-relay"
+description = "Implementation of a `https://tanssi.network` node in Rust based on the Substrate framework."
+edition = "2021"
+license = "GPL-3.0-only"
+readme = "README.md"
+rust-version = "1.64.0"
+version = "0.1.0"
+
+# Configuration for building a .deb package - for use with `cargo-deb`
+[package.metadata.deb]
+name = "tanssi-relay"
+assets = [
+	[
+	"/usr/bin/",
+	"755",
+	"target/release/tanssi-relay",
+],
+	[
+	"/usr/lib/tanssi-relay/",
+	"755",
+	"target/release/tanssi-relay-prepare-worker",
+],
+	[
+	"/usr/lib/tanssi-relay/",
+	"755",
+	"target/release/tanssi-relay-execute-worker",
+],
+	[
+	"/lib/systemd/system/",
+	"644",
+	"scripts/packaging/tanssi-relay.service",
+],
+]
+conf-files = [ "/etc/default/tanssi-relay" ]
+extended-description = "Implementation of a https://tanssi.network node in Rust based on the Substrate framework."
+license-file = [ "0", "LICENSE" ]
+maintainer = "security@tanssi.network"
+# https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html
+maintainer-scripts = "scripts/packaging/deb-maintainer-scripts"
+section = "misc"
+[[bin]]
+name = "tanssi-relay"
+path = "src/main.rs"
+
+[[bin]]
+name = "tanssi-relay-execute-worker"
+path = "src/bin/execute-worker.rs"
+
+[[bin]]
+name = "tanssi-relay-prepare-worker"
+path = "src/bin/prepare-worker.rs"
```

Comment:
We probably don't need these to be in our repo, we can just use the binaries from the polkadot repo, which moonwall already downloads. So can you try removing these files and see if tests keep working?

Reply:
I ran into an issue:  `moonwall` downloads them it's in `x86` architecture and I only have `Apple Silicon` so locally it's causing issues as I can't run moonwall tests. However, suspect this is isolated issue.

Long term solution could be to have the upstream cross-compile them so moonwall can check architecture and download accordingly.

I will remove before merging the PR.